dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1539,11 +1539,28 @@ ENTITY_NOT_SUPPORTED_CNTR0335W.explanation=Entity enterprise beans declared in a
-NAME_ALREADY_BOUND_WARN_CNTR0336W=CNTR0336W: The {0} interface of the {1} bean in the {2} module of the {3} application cannot be bound to the {4} name location. The {5} interface of the {6} bean in the {7} module of the {8} application is already bound to the {4} name location. The {4} name location is not accessible.
-NAME_ALREADY_BOUND_WARN_CNTR0336W.explanation=Each enterprise bean interface must specify a unique binding location.
-NAME_ALREADY_BOUND_WARN_CNTR0336W.useraction=Correct the binding file for one of these two bean interfaces so that each interface has a unique binding location.
+#
+# CNTR0336W: Already in use.
+# CNTR0337W: Already in use.
+#
+#-------------------------------------------------------------------------------
 
+#-------------------------------------------------------------------------------
+NAME_ALREADY_BOUND_WARN_CNTR0338W=CNTR0338W: The {0} interface of the {1} bean in the {2} module of the {3} application cannot be bound to the {4} name location. The {5} interface of the {6} bean in the {7} module of the {8} application is already bound to the {4} name location. The {4} name location is not accessible.
+NAME_ALREADY_BOUND_WARN_CNTR0338W.explanation=Each enterprise bean interface must specify a unique binding location.
+NAME_ALREADY_BOUND_WARN_CNTR0338W.useraction=Correct the binding file for one of these two bean interfaces so that each interface has a unique binding location.
 
+#-------------------------------------------------------------------------------
+# Do NOT translate "JNDI", "namespace", or "ejb-jar-bnd.xml".
+NAMESPACE_IN_JNDI_BINDING_NAME_CNTR0339E=CNTR0339E: The {2} Java Naming and Directory Interface (JNDI) binding name for the {0} bean or home interface in the {1} module contains a namespace.
+NAMESPACE_IN_BINDING_NAME_CNTR0339E.explanation=JNDI binding names must not contain a namespace, which is denoted by the ':' character. The entire string before and including the ':' character is considered a namespace.
+NAMESPACE_IN_BINDING_NAME_CNTR0339E.useraction=Correct the ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the namespace.
+
+#-------------------------------------------------------------------------------
+# Do NOT translate "JNDI", "namespace", "ejb-jar-bnd.xml", or "ejblocal:".
+NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340E=CNTR0340E: The {2} Java Naming and Directory Interface (JNDI) binding name for the {0} bean or local home interface in the {1} module contains an invalid namespace.
+NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340E.explanation=The local binding name can only contain one namespace and it must start with ejblocal:. Namespaces are denoted by the ':' character. The entire string before and including the ':' character is considered a namespace.
+NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340E.useraction=Correct the ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the invalid namespace.
```

Comment:
**change**
Correct the ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the invalid namespace.
**to**
Correct the ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the namespace that is not valid.